js/crm.backdrop.js

```diff
@@ -0,0 +1,14 @@
+// http://civicrm.org/licensing
+CRM.$(function($) {
+  $(document).on('crmLoad', function(e) {
```

Comment:
You don't need this wrapper. We're already waiting for document.ready because of the `CRM.$(function()...` construct